src/VerifyDuoController.js

```diff
@@ -122,7 +122,7 @@ export default FormController.extend({
     },
   },
 
-  Footer: FooterSignout,
+  Footer: FooterMFA,
```

Comment:
Looks like in `VerifyDuo`, `VerifyU2F`, `VerifyWebauthn`, `VerifyWindowsHello`
Footer was added without checking feature `features.hideSignOutLinkInMFA` and `features.mfaOnlyFlow`.
@gowthamidommety-okta @mauriciocastillosilva-okta do you know if this is intentional or bug?

Reply:
Seems like something we should fix. The only reason I can think of for why we did not add this feature here, is that these factors may not be used for mfa only flows. However, I don't see any harm in adding a check for this feature. In fact, i feel like the feature would be more complete if we added it. @mauriciocastillosilva-okta thoughts?

Reply:
Sounds right, we probably missed those. The original change for `hideSignOutLinkInMFA` only covered the generic `MFAVerifyController`. 
So +1, it is a bug, since all factors should behave the same about show/hide the footer.

Reply:
Thank you @gowthamidommety-okta @mauriciocastillosilva-okta !

@denysoblohin-okta we could
- either fix in this PR
- open another PR to fix this inconsistent first 

Then your PR would looks a little clear (e.g. instead of passing `showLink`, we can check the feature flag within `FooterMFA`)

Reply:
@haishengwu-okta https://github.com/okta/okta-signin-widget/pull/1635